variables.tf

```diff
@@ -11,6 +11,14 @@ variable "cluster_name" {
   type = string
 }
 
+variable "cluster_endpoint_public_access" {
+  type = bool
+}
+
+variable "cluster_endpoint_public_access_cidrs" {
+  type = list(string)
```

Comment:
Same here. The default should probably be `["0.0.0.0/0"]` and can be overridden if desired.